src/components/Footer.js

```diff
@@ -0,0 +1,45 @@
+import React, { Component } from 'react';
+import TwitterSVG from '../svg/Twitter';
+import FacebookSVG from '../svg/Facebook';
+import GithubSVG from '../svg/Github';
+import ExternalLinkSVG from '../svg/ExternalLink';
+import '../css/components/Footer.css';
+
+class Footer extends Component {
+
+  render() {
+    return (
+      <footer className="coa-Footer">
+        <div className="coa-section row">
```

Comment:
since coa-section has margins on it, we can't use row in the same element, since row has conflicting margins. I know it'll result in a lot of nesting, so I'm open to suggestions as to how to handle this better.

Reply:
hmmm, nesting isn't the worst at this point so I'll go with that.